takwimu/urls.py

```diff
@@ -20,6 +20,8 @@
                   url(r'^profiles/$', ProfileView.as_view(), name='profiles'),
                   url(r'^topics/$', TopicView.as_view(), name='topics'),
                   url(r'^feed/$', CountryProfileFeed(), name='rss_feed'),
+                  # testing search page
+                  url(r'^search$', search_view, name='search'),
```

Comment:
Shouldn't the search URL also end in  `/`? To keep it uniform with the rest if nothing else.